docs/UserGuide.md

```diff
@@ -1225,7 +1224,7 @@ This section describes commands tht help you manage your students' graded tests.
 
 #### 🛠️Editing a Graded Test Score: `editgradedtest`
 
-You can edit your student’s graded test scores.
+You can edit your student's graded test scores if there is a need to make adjustments or corrections to the initially recorded test scores.
```

Comment:
You can edit your student's graded test scores if `you` need to make adjustments or corrections to their initially recorded test scores.